content/collections/guides_and_surveys/en/resource-center-autopilot.md

```diff
@@ -0,0 +1,15 @@
+---
+id: e5b38789-2785-4fcc-bff1-2c50979c3546
+blueprint: guides_and_survey
+title: Autopilot
+landing: false
+updated_by: 15756874-6eaf-4a8c-8779-bd4081ba41b6
+updated_at: 1750961197
+this_article_will_help_you:
+  - 'Understand the smart Autopilot features for Resource Center'
+---
+Autopilot for Resource Center recommendation sets automatically surfaces useful articles to your users depending on where they are in your website. It provides a smart fallback if no recommendation sets have been created for any specific page. By using Autopilot, you don't have to manually create recommendation sets yourself. Autopilot pulls content from your repositories. This reduces the amount of setup and maintenance time for the Resource Center as a whole. 
```

Comment:
@JeffScattini-amplitude Any reason for this being a standalone article, and not placing this content in Settings?